Makefile

```diff
@@ -645,10 +644,6 @@ FILES_TO_CLEAN+=install/consul/istio.yaml \
                 install/kubernetes/istio-auth.yaml \
                 install/kubernetes/istio-ca-plugin-certs.yaml \
                 install/kubernetes/istio-ca-with-health-check.yaml \
```

Comment:
@wattli  Can we remove some of the 'ca-with' yaml - and use helm options instead ? We expect most people to use 'helm' or 'helm template' - without custom yaml files.